packages/react-wrapper/package.json

```diff
@@ -40,15 +40,15 @@
     "vue": "^2.6.0"
   },
   "devDependencies": {
-    "@types/jest": "~25.2.3",
+    "@types/jest": "~27.0.3",
     "@types/react": "~16.9.0",
     "@types/react-dom": "~16.9.0",
     "cypress": "~8.4.0",
     "eslint-plugin-react": "~7.20.0",
-    "jest": "~26.0.1",
-    "parcel": "~1.12.4",
+    "jest": "~27.3.1",
+    "parcel": "~2.0.1",
```

Comment:
I had to update parcel because of that error

![image](https://user-images.githubusercontent.com/3502075/143185672-fa54f6b0-b1f3-4c8f-a18f-fb35adfb3e8c.png)

Reply:
it seems a bug with babel: 
https://github.com/parcel-bundler/parcel/issues/5943

it's ok the update though

Reply:
It seems because of a bug with babel: https://github.com/parcel-bundler/parcel/issues/5943

it's ok the update though